runtime/stdlib/account.go

```diff
@@ -2256,7 +2407,109 @@ func newPublicAccountCapabilitiesValue(
 	return interpreter.NewPublicAccountCapabilitiesValue(
 		gauge,
 		addressValue,
-		nil,
-		nil,
+		newAccountCapabilitiesGetFunction(gauge, addressValue, sema.PublicAccountCapabilitiesTypeGetFunctionType, false),
+		newAccountCapabilitiesGetFunction(gauge, addressValue, sema.PublicAccountCapabilitiesTypeBorrowFunctionType, true),
+	)
+}
+
+func newAccountCapabilitiesGetFunction(
+	gauge common.MemoryGauge,
+	addressValue interpreter.AddressValue,
+	funcType *sema.FunctionType,
+	borrow bool,
```

Comment:
Having to pass both the condition ('borrow' flag) and the type can be a little error-prone. Instead, maybe, either:
 - Use the `funcType` to determine whether it's a borrow or a get.
 - Or, maybe even better, accept only the 'borrow' flag, and pick the function type depending on the flag
   ```go
   if borrow {
       type = sema.PublicAccountCapabilitiesTypeBorrowFunctionType
   } else {
       type = sema.PublicAccountCapabilitiesTypeGetFunctionType
   }
   ```

Reply:
Good idea, improved it in af6282365b4e90f5c6b7622d91ff6139927ab3a1